### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by belokonm** [FluidTYPO3/fluidcontent_core#133]
Hi guys. 
Can you share any link / video to see how to use this ext?
I did 
1. Disable the extension "CSS styled content" (css_styled_content). You can't use css_styled_content and fluidcontent_core at the same time!
2. Download the extension and install it.
3. Copy, or integrate into your own, the AdditionalConfiguration.php file from EXT:fluidcontent_core/Build/AdditionalConfiguration.php into typo3conf/AdditionalConfiguration.php - or use the Extension Manager upgrade script for fluidcontent_core which does this for you.
4. Load the static TypoScript template to make add the configuration options which define the CSS class names you can set on container HTML elements, the default H-size, and more.

After all my content is disappear from FE. 
In my template for content showing I'm using next: <div class="right"><f:format.raw>{content}</f:format.raw></div>

In TS:
page.10 = FLUIDTEMPLATE
page.10 {
    format = html
    file = fileadmin/templates/main_template.html
    variables {
        content_header < styles.content.get
        content_header.select.where = colPos = 2
        content_slogan< styles.content.get
          content_slogan.select.where = colPos = 1
        content< styles.content.get
          content.select.where = colPos = 0
    }
}

Did I mist something?

**Comment by NamelessCoder** [comment]
The TypoScript you are setting up seems to depend on CSS styled content. If you replicated the TypoScript for `styles.content.get` manually then you can render those from Fluid by referencing the object path in this ViewHelper:

https://fluidtypo3.org/viewhelpers/fluid/master/CObjectViewHelper.html

If the TypoScript is copy/pasted from another place where CSS styled content is assumed then you need to remove those variable assignments and instead use `v:content.render` with argument `column` to set the desired colPos:

https://fluidtypo3.org/viewhelpers/vhs/master/Content/RenderViewHelper.html

Basically, `fluidcontent_core` does not support the TypoScript from `css_styled_content` as it renders everything using Fluid, not TypoScript. It's the key difference between the two extensions ;)

https://fluidtypo3.org/viewhelpers/fluid/master/CObjectViewHelper.html
